go/vt/wrangler/tablet_test.go

```diff
@@ -82,17 +76,14 @@ func TestDeleteTabletBasic(t *testing.T) {
 		Keyspace: "test",
 	}
 
-	if err := wr.TopoServer().InitTablet(context.Background(), tablet, false /*allowPrimaryOverride*/, true /*createShardAndKeyspace*/, false /*allowUpdate*/); err != nil {
-		t.Fatalf("InitTablet failed: %v", err)
-	}
+	err := wr.TopoServer().InitTablet(context.Background(), tablet, false /*allowPrimaryOverride*/, true /*createShardAndKeyspace*/, false /*allowUpdate*/)
+	require.NoError(t, err, "InitTablet failed")
 
-	if _, err := ts.GetTablet(context.Background(), tablet.Alias); err != nil {
-		t.Fatalf("GetTablet failed: %v", err)
-	}
+	_, err = ts.GetTablet(context.Background(), tablet.Alias)
+	require.NoError(t, err, "GetTablet failed")
 
-	if err := wr.DeleteTablet(context.Background(), tablet.Alias, false); err != nil {
-		t.Fatalf("DeleteTablet failed: %v", err)
-	}
+	err = wr.DeleteTablet(context.Background(), tablet.Alias, false)
+	require.NoError(t, err, "DeleteTablet failed")
```

Comment:
👍

Reply:
The `"DeleteTablet failed"` bit can be removed here I think?